tests/scripts/all.sh

```diff
@@ -2318,11 +2357,9 @@ component_test_psa_crypto_config_accel_ecdh () {
 
```

Comment:
I don't think this satisfies the test requirements in https://github.com/Mbed-TLS/mbedtls/issues/8016. Specifically:

> at least test one case where there's one accelerated curve and one built-in curve, one case where ECDH is accelerated but not ECDSA, and one case where it's the other way round.

for which [Manuel proposed a strategy](https://github.com/Mbed-TLS/mbedtls/issues/8016#issuecomment-1663446576).

Reply:
I thought that for the 1st part ("at least test one case where there's one accelerated curve and one built-in curve") the following components were enough:
- `test_psa_crypto_config_accel_all_curves_except_p192`
- `test_psa_crypto_config_accel_all_curves_except_x25519`

and that for the 2nd one ("one case where ECDH is accelerated but not ECDSA, and one case where it's the other way round") these were enough:
- `test_psa_crypto_config_accel_ecdh`
- `test_psa_crypto_config_accel_ecdsa`

Or you were asking for a more detailed scenario in which both 1st and 2nd case were mixed?
I mean, something like:
- `accel_all_curves_except_p192` + accelerate ECDH only
- `accel_all_curves_except_p192` + accelerate ECDSA only
- `accel_all_curves_except_x25519` + accelerate ECDH only
- `accel_all_curves_except_x25519` + accelerate ECDSA only
?

Reply:
Thanks for the breakdown, I might have missed some of those in my review. I'll re-check.

Reply:
Having re-checked, I'm satisfied that the existing components provide enough coverage. Sorry for the noise.

Reply:
No problem at all! Thanks for the review :)